packages/pigeon/pigeons/primitive.dart

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:pigeon/pigeon.dart';
+
+@HostApi()
+abstract class PrimitiveHostApi {
+  int inc(int value);
+  bool aBool(bool value);
+}
+
+@FlutterApi()
+abstract class PrimitiveFlutterApi {
+  int inc(int value);
```

Comment:
Why the strange abbreviation here?

Reply:
TLDR Done

"inc" is a typical function found in other (old) languages that means "increment".  I was going to have this function do something but then decided that echoing is enough.